test/extended/operators/olm.go

```diff
@@ -70,4 +70,30 @@ var _ = g.Describe("[Feature:Platform] OLM should", func() {
 			}
 		})
 	}
+
+	// OCP-20981, [BZ 1626434]The olm/catalog binary should output the exact version info
+	// author: jiazha@redhat.com
+	g.It("olm version should contain the source commit id", func() {
+		catalogPodName, err := oc.AsAdmin().Run("get").Args("-n", "openshift-operator-lifecycle-manager", "pods", "-l", "app=catalog-operator", "-o=jsonpath={.items[0].metadata.name}").Output()
+		o.Expect(err).NotTo(o.HaveOccurred())
+		e2e.Logf("catalog-operator pod name:%s", catalogPodName)
```

Comment:
I believe we should check version of the `olm operator` as well. Does the `package-server` output version as well?
Also, we want to verify that all three components report the same version and commit sha? ( I am assuming that we are cutting a build for all three components at the same point in time )

Reply:
> I believe we should check version of the olm operator as well.

I think we should.
 
> Does the package-server output version as well?

No, it does not.

> Also, we want to verify that all three components report the same version and commit sha?

I think we should.

> ( I am assuming that we are cutting a build for all three components at the same point in time )

We are.

Reply:
Thanks! I will add the `olm operator` checking.

Reply:
Done.